components/staking/TableValidators.vue

```diff
@@ -1,33 +1,23 @@
 <template>
-  <div class="container">
-    <table v-if="showingValidators.length" class="data-table">
-      <thead>
-        <PanelSort
-          :sort="sort"
-          :properties="properties"
-          :show-on-mobile="showOnMobile"
-        />
-      </thead>
-      <tbody
-        v-infinite-scroll="loadMore"
```

Comment:
This is broken now. Not all validators are loaded and scrolling down doesn't trigger anything. Just shows 25 validators.

Fixing it